src/app.ts

```diff
@@ -178,6 +186,32 @@ app.put(
   )
 );
 
+app.put(
+  "/services/:serviceId/logo",
+  ouathVerifier,
+  wrapRequestHandler(
+    withRequestMiddlewares(
+      getApiClientMiddleware(),
+      getUserFromRequestMiddleware(),
+      RequiredParamMiddleware("serviceId", NonEmptyString),
```

Comment:
Can we use `ServiceId` as type instead of `NonEmptyString` ?

Reply:
thanks